backends/dpdk/dpdkArch.cpp

```diff
@@ -3035,4 +3035,169 @@ std::vector<std::pair<cstring, const IR::Type *>>
 cstring DpdkAddPseudoHeaderDecl::pseudoHeaderInstanceName;
 cstring DpdkAddPseudoHeaderDecl::pseudoHeaderTypeName;
 
+void InsertReqDeclForIPSec::uniqueNames() {
```

Comment:
can we make it a utility? taking vector of cstring and refMap as arg?

Reply:
Done